fcc_submitter.py

```diff
@@ -153,108 +153,6 @@ def __init__(self, from_prj, to_prj, submit_limit):
-                'tulip',
-                'guake',
-                'mlterm',
-                'uim',
-                '^libxml',
-                'w3m-el',
-                'scim$',
-                '^scim-(\w+)',
-                'gstreamer-0_10-plugins-gl',
-                'libgdamm',
-                'gtk3-metatheme-sonar',
-                'gstreamer-0_10-plugin-crystalhd',
-                'grisbi',
-                'heroes-tron',
-                'specto',
-                'wayland-protocols',
-                'gsf-sharp',
-                'hal-flash',
-                'kdelibs3',
-                'qca-sasl',
-                'mozaddon-gnotifier',
-                'khunphan',
-                'lxcfs',
-                'containerd',
-                'docker-bench-security',
-                '0ad-data',
-                'python-plaso',
-                'gnome-news',
-                'wdm',
-                'nuntius',
-                'gobby04',
-                'sobby',
-                'efax-gtk',
-                'gnome-blog',
-                'clipit',
-                'fyre',
-                'gDesklets',
-                'texi2html',
-                'texi2roff',
-                'typo3-cms-6_2',
-                'decibel-audio-player',
-                'jamin',
-                'twind',
-                'ksshaskpass',
-                'sdl-asylum',
-                'nepomuk-core',
-                'audaspace',
-                'e_dbus',
-                'gnome-do',
-                'giver',
-                'php5-APCu',
-                'php5-memcache',
-                'java-1_9_0-openjdk',
-                'python-pandas',
-                '^bundle-lang',
-                'docker-image-migrator',
-                'metacity-themes',
-                'gnome-python-desktop',
-                'kiwi-config-openSUSE',
-                'tecnoballz',
-                'zaz',
-                'sawfish-pager',
-                'libgepub',
-                '^akonadi',
-                '^ffmpeg',
-                'sdl_bomber',
-                'double-cross',
-                'aop',
-                'viruskiller',
-                'ceferino',
-                'sdlscavenger:',
-                'asteroid',
-                'xgalaga-sdl',
-                'libgig',
-                'blog',
-                'mt-st',
-                'lostfeathers',
-                'gstreamer-rtsp-server',
-                'libgnome-games-support',
-                'php5-ZendFramework',
-                'runc',
-                'rubygem-json_pure-1_5',
-                'rubygem-addressable-2_3',
-                'rubygem-arel-6',
-                'rubygem-colorator-0',
-                'rubygem-listen-3_0',
-                'rubygem-rouge-1',
-                'rubygem-json-schema-2_2',
-                'computertemp',
-                'libtiger',
-                ]
         self.check_later = [
                 'tulip',
                 'khunphan',
```

Comment:
hehe, check_later is used nowhere in this tool, just remind myself, but can be add to skip list too since the maintainer should submit them when they think it's ready.